Report/Chapters/Introduction.tex

```diff
@@ -2,7 +2,6 @@ \chapter{Introduction}
 
 Programming has always been an activity which is done on a computer with a keyboard. Despite the existence of different visual programming languages, like Blender, there are no programs in which well known programming languages like Java, Haskell or C(++) can be visually implemented. 
 
-In this project we have (attempted) to make an application in which the programming language Haskell can be visually implemented.
-%aanvullen als het project af is
+In this project we have (attempted) to make an application in which the programming language Haskell can be visually implemented. We named our approach ``Viskell''. Viskell tries to aide developers with real-time feedback on Haskell programs. This allows developers to spot type errors directly, without needing to have a compilable program, and provides a visual insight into the program. Also, the intermediate output of a function can be inspected at runtime. Viskell also allows developers to collaborate and work together on a large multi-touch screen. Other common touch interfaces as found on devices like the Microsoft Surface (Pro), are supported as well. Because of the user interface with the touch capabilities developers can easily manipulate the code without having a keyboard or mouse around, so programming on the go is a better experience. Besides that programming with Viskell should be more fun and more ergonomical because seating positons can be changed.
```

Comment:
> so programming on the go is a better experience.

Not so sure about that tbh

The entire section has lots of claims that might not be fully justified